docs/source/intervention_models/maternal_anemia_intervention/index.rst

```diff
@@ -0,0 +1,248 @@
+.. _maternal_anemia_intervention:
+
+===================================
+Tiered Maternal Anemia Intervention
+===================================
+
+.. contents::
+   :local:
+   :depth: 1
+
+.. list-table:: Abbreviations
+  :widths: 10 10
+  :header-rows: 1
+
+  * - Abbreviation
+    - Definition
+  * - BEP
+    - Balanced Energy Protein
+  * - DFS
+    - Double Fortified Salt
+  * - MMS
+    - Multiple Micronutrient Supplement
+  * - IFA
+    - Iron-folic Acid
+  * - ANC
+    - Antenatal Care
+  * - BMI
+    - Body Mass Index
+  * - PLW
+    - Pregnant and Lactating Women
+  * - LMICs
+    - Low- and Middle-Income Countries
+  * - Focal countries
+    - Pakistan, Bangladesh, Nigeria, Malawi
+
+Intervention Overview
+---------------------
+Tiered maternal anemia intervention is an integrated life course intervention 
+designed for at-risk female popualtion (e.g., women during pregnancy, women of 
+reprodictive age, etc.) for averting the burden of maternal anemia in LMICs. 
+It consists of three tiers: 1) Large-scale food fortificaiton (DFS), 2) Supplmentation 
+(BEP and MMS), and 3) treatment (IV iron).
+
+.. image:: intervention_pyramid.png
+
+The scope of our simulation work is to estimate following measures in 4 focal countries:
+ - the size of eligible population at risk for each tier of the pyramid;
+ - the burden of the maternal anemia in terms of deaths and DALYs occurring in 
+   the population at risk within each tier of the pyramid if we assume 
+   intervention is not applicable; 
+ - the impact at each tier on maternal anemia in terms of averted deaths and DALYs 
+   if we assume complete pyramid of interventions are applied.
+
+.. list-table:: Tiered Intervention
+  :header-rows: 1
+
+  * - Tier
+    - Intervention
+    - Product name
+    - Product definition
+    - Target population
+    - Note
+  * - 1
+    - Large-scale food fortification
+    - DFS
+    - 
+    - Women of reproductive age (15 to 49 years)
+    - 
+  * - 2
+    - Supplementation
+    - BEP
+    - 
+    - PLW at ANC service with BMI less than 18.5
+    - 
+  * - 2
+    - Supplementation
+    - MMS
+    - 
+    - PLW at ANC service with BMI greater than or equal to 18.5
+    - 
+  * - 3
+    - Treatment
+    - IV iron
+    - 
+    - Pregnant women in 2nd/3rd trimester with moderate/severe anemia referred 
+      from ANC to higher level facility
+    - 
+
+.. todo::
+
+  Fill out the table above with intervention product definition.
+
+.. list-table:: Affected Outcomes
+  :header-rows: 1
+
+  * - Outcome
+    - Effect
+    - Population
+    - Note
+  * - Population mean hemoglobin
+    - Additive
+    - non-PLW
+    - Effects are not stratified by intervention tiers
+  * - Population mean hemoglobin
+    - Additive and multiplicative combined
+    - PLW
+    - Effects are stratified by intervention tiers 
+
+Baseline Coverage Data
+++++++++++++++++++++++
+
+.. note::
+
+  No existing coverage of each intervention tier
```

Comment:
I think there is existing coverage of iron supplementation among PLW. Perhaps link to the maternal supplementation page that has information on this.

Reply:
I don't have materials about the treatment coverage among target populations, can i find it out in J drive?

Reply:
It's listed in the baseline coverage section here: https://vivarium-research.readthedocs.io/en/latest/intervention_models/targeted_bep_and_mms/index.html

I believe we may have ignored baseline coverage for this project at the time due to the rush nature, but we should consider it going forward!

Reply:
I see, thanks!